client/lib/akismet/is-akismet-checkout.ts

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
FYI this is being added in https://github.com/Automattic/wp-calypso/pull/76493 as well